src/Models/ApplicantModel.php

```diff
@@ -438,6 +438,22 @@ public function updateApplicantAdditionalFields(array $applicant)
         return $query->execute();
     }
 
+    public function addApplicantPassword(string $password, int $applicantId)
+    {
+        $sql = 'UPDATE `applicants` SET `profile_password` = :password WHERE `id` = :applicantId';
+        $query = $this->db->prepare($sql);
+        return $query->execute([':password' => $password, ':applicantId' => $applicantId]);
+    }
+
+
+    public function getApplicantPassword(int $applicantId)
+    {
+        $sql = 'SELECT `profile_password` FROM `applicants` WHERE `id` = :applicantId';
+        $query = $this->db->prepare($sql);
+        $query->execute([':applicantId' => $applicantId]);
+        return $query->fetch();
```

Comment:
this returns an array, but could just return a single value. Look up PDO FETCH_COLUMN